### PR TITLE
Add prepublishing, linting, test-coverage

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# ./node_modules/.bin on the PATH
+export PATH := ./node_modules/.bin:$(PATH)
+
+# Use bash not sh
+SHELL := /bin/bash
+
+# Environment variables
+# ---------------------
+
+EXPECTED_COVERAGE = 50
```
Comment (JakeChampion) [inline Makefile:10]: We should look to improve this coverage in another PR

Comment (JakeChampion) [reply Makefile:10]: https://github.com/Financial-Times/get-github-public-organisation-repositories/issues/4

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  override:
+    - make verify
+    - make test
+
+deployment:
+  release:
+    tag: /v.*/
+    owner: Financial-Times
+    commands:
+      - make npm-publish
```
Comment (rowanmanning) [inline circle.yml]: This currently requires you to add additional commands into the Circle UI (which I'd like to switch all of our modules away from). A better example to follow is: https://github.com/Financial-Times/origami-service-makefile/blob/master/circle.yml

**Review by rowanmanning — COMMENTED** [review]
This looks great 👍 small suggested change to the Circle config (and we'll need to add environment variables to Circle too)

**Review by rowanmanning — APPROVED** [review]

**Pull request merged**